src/historywork/CheckSingleLedgerHeaderWork.h

```diff
@@ -0,0 +1,35 @@
+#pragma once
+
+// Copyright 2021 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+#include "history/FileTransferInfo.h"
```

Comment:
Many of these includes (HistoryArchive, GetAndUnzipRemoteFileWork, TmpDir) can be forward declared

Reply:
Fixed.